src/praisonai/praisonai/bots/_commands.py

```
@@ -775,3 +784,217 @@ def handle_learn_command(
+    Args:
+        session_manager: BotSessionManager instance.
+        user_id: User ID issuing the command.
+
+    Returns:
+        The previous user message to re-run, or guidance.
+    """
+    storage_key = user_id
+    if hasattr(session_manager, "_storage_key"):
+        try:
+            storage_key = session_manager._storage_key(user_id)
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)